roles/ffh.yanic/defaults/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 
 go_version: "1.13.1"
+go_bin_path: "/usr/local/go/bin/"
+go_path: "/opt/go/"
```

Comment:
Paths don't need trailing slash

Reply:
Not if used properly; but in case someone concats a path imporperly, it might break things.
I've got no strong opinion on this. It was a conscious decision. But if this has nasty sideeffects, we might change it?

Reply:
If a double slash is no problem...